templates/commands/render_test.go

```diff
@@ -536,11 +557,11 @@ func TestSafeRelPath(t *testing.T) {
 
 			got, err := safeRelPath(nil, tc.in)
 
-			if got != tc.want {
-				t.Errorf("safeRelPath(%q)=%q, want %q", tc.in, got, tc.want)
+			if got, want := got, tc.want; got != want {
+				t.Errorf("safeRelPath(%s): expected %q to be %q", tc.in, got, want)
```

Comment:
I got really confused by the error output, so I changed this

Reply:
SGTM